doc/repoquery.rst

```diff
@@ -55,7 +55,7 @@ Together with ``<pkg-spec>``, control what packages are displayed in the output.
     Limit the resulting set to leaves packages that were installed as dependencies so they are no longer needed. This switch lists packages that are going to be removed after executing ``dnf autoremove`` command.
 
 ``--available``
-    Limit the resulting set to available packages only.
+    Limit the resulting set to available packages only (set by default).
```

Comment:
and what about only (default) ? I am enabling of --installed with --available, therefore it is still handy.